### PR TITLE
Added Indonesian Translation

### PR DESCRIPTION
**Pull request by MuzTank**
Added Indonesian Translation

### CODE REVIEW

**Comment by javuto** [comment]
Created issue here: https://github.com/facebook/fbctf/issues/250
Thank you!

File: src/language/lang_id.php [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+    'D, d-m-Y H:i:s', //used by date() function
+  //Translations for IndexController
+  'Facebook CTF' =>
+    'Facebook CTF',
+  'Conquer the world' =>
+    'Kuasai dunia',
+  'Play' =>
+    'Mulai',
+  'Welcome to the Facebook Capture the Flag Competition. By clicking "Play," you will be entered into the official CTF challenge. Good luck in your conquest.' =>
+    'Selamat datang di Kompetisi Facebook Capture the Flag. Dengan menekan "Mulai" maka kamu akan dapat mulai bermain di Kompetisi Facebook CTF. Semoga beruntung dalam menguasai dunia.',
```
Comment (juliannagler) [inline src/language/lang_id.php]: Suggested change: 'Selamat datang di Kompetisi Facebook Capture the Flag. Dengan menekan "Main" maka kamu akan dapat mulai bermain di Kompetisi Facebook CTF. Semoga beruntung dalam menguasai dunia.'

Comment: Mistranslation of the term "Play" into "Start"

File: src/language/lang_id.php [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+  'Are you sure you want to logout from the game?' =>
+    'Apakah kamu yakin ingin keluar dari permainan?',
+  'Saved' =>
+    'Tersimpan',
+  'All changes have been successfully saved.' =>
+    'Semua perubahan telah berhasil disimpan.',
+  'Error' =>
+    'Error',
+  'Sorry your form was not saved. Please correct the all errors and save again.' =>
+    'Gagal menyimpan. Mohon perbaiki semua erro yang terjadi dan cobalah menyimpan kembali.',
```
Comment (juliannagler) [inline src/language/lang_id.php]: Suggested Change: 'Formulir kamu tidak disimpan.'

Comement: Context mistranslation of the sentence. It seemed that the form was not saved due to technical reasons."Gagal menyimpan" = "Fail to save".

File: src/language/lang_id.php [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+  'cancel_' =>
+    'batal_',
+  'Are you sure you want to cancel? You have unsaved changes that will be reverted.' =>
+    'Apakah kamu yakin ingin membatalkan? Semua perubahan yang terjadi tidak akan disimpan.',
+  'choose_logo' =>
+    'pilih_logo',
+  'captured_' =>
+    'captured_',
```
Comment (juliannagler) [inline src/language/lang_id.php]: Suggested Change: 'menguasai'

File: src/language/lang_id.php [hunk 4]
```diff
@@ -0,0 +1,659 @@
...
+  'category' =>
+    'kategori',
+  'capture_' =>
+    'capture_',
+  'Insert your answer' =>
+    'Masukkan jawaban',
+  'Request Hint' =>
+    'Minta Petunjuk',
+  'Submit' =>
+    'Masukkan',
```
Comment (juliannagler) [inline src/language/lang_id.php]: Suggested Change: 'Kirim'

Comment: "Submit" mistranslated into "Insert".

File: src/language/lang_id.php [hunk 5]
```diff
@@ -0,0 +1,659 @@
...
+  'Game_On' =>
+    'Game_On',
+  'tutorial_' =>
+    'tutorial_',
+  'Next' =>
+    'Selanjutnya',
+  'Skip to play' =>
+    'Langsung bermain',
+  'Powered By Facebook' =>
+    'Dibuat Oleh Facebook',
```
Comment (juliannagler) [inline src/language/lang_id.php]: Suggested Change: 'Didukung oleh Facebook'

Comment: "Powered by" mistranslated into "Made by".

**Comment by juliannagler** [comment]
Adding in full string dump with updated changes. 

[Indonesian FBCTF Translation - Revised - CLEAN.docx](https://github.com/facebook/fbctf/files/570133/Indonesian.FBCTF.Translation.-.Revised.-.CLEAN.docx)

**Comment by MuzTank** [comment]
Pull request updated 931b319a9ab824f0bd5694988ffce0d6051a590d
Sorry, just have time to change that.

**Comment by juliannagler** [comment]
Not a problem, thanks @MuzTank! As you use the translated platform, please be sure to file issues for any formatting problems you run into. 

Awesome work!

**Pull request merged**